cli/mizarapi.py

```diff
@@ -36,13 +36,13 @@ def create_net(self, name, ip, prefix, vpc, bouncers=1, vni=None):
             "vpc": vpc,
             "bouncers": bouncers
         }
-        self.create_obj(name, "Net", spec, "nets")
```

Comment:
It's better to split the renaming part as a separate PR.   It will make review easier and commits more portable.

Reply:
Pod create move to https://github.com/futurewei-cloud/mizar/pull/224
I only created PR for pod create (not change naming to net to subnet) is because part of the net to subnet change is based off of pod create PR. I am hoping to merge my pod create PR first before I create PR for net to subnet change

Reply:
Is this PR still valid? I see there are lots of duplication between this PR and your separated PR.

Reply:
NO, will close this one